### PR TITLE
improve texts of fretboard diagram inspector and add acessibillity texts to reset inspector's buttons

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
* improve texts of fretboard diagram inspector and a fix for a compiler warning
* add accessibility texts to Inspector's reset buttons

### CODE REVIEW

File: mscore/inspector/inspector_fret.ui [hunk 1]
```diff
@@ -237,11 +237,7 @@
        </widget>
       </item>
       <item row="6" column="2">
-       <widget class="Ms::ResetButton" name="resetShowNut" native="true">
-        <property name="accessibleName">
-         <string>Reset show nut</string>
```
Comment (jthistle) [inline mscore/inspector/inspector_fret.ui]: why remove the accessible name for this?

Comment (Jojo-Schmitz) [reply mscore/inspector/inspector_fret.ui]: Good question: because no other reset button in any inspector ui has them anymore.

Comment (jthistle) [reply mscore/inspector/inspector_fret.ui]: OK, fair enough. The latter option would be ideal but I can see that that's not something for this PR :)

Comment (Jojo-Schmitz) [reply mscore/inspector/inspector_fret.ui]: Yes, separate PR or separate commit to this here

**Comment by Jojo-Schmitz** [comment]
As there would possibly be merge conflicts if done in separate PRs, I'll add a commit with accessibility texts to this PR here. Unless there are objections, @anatoly-os ?

**Comment by Jojo-Schmitz** [comment]
These are quite a few new texts, so an early merge is appreciated to give the translators enough time.

File: mscore/fretproperties.cpp [hunk 2]
```diff
@@ -343,7 +343,7 @@ void FretCanvas::mousePressEvent(QMouseEvent* ev)
                               };
 
                               // Find the lowest dot type that doesn't already exist on the string
-                              for (int i = 0; i < dtypes.size(); i++) {
+                              for (int i = 0; i < int(dtypes.size()); i++) {
```
Comment (jthistle) [inline mscore/fretproperties.cpp]: I know std::vector::size doesn't necessarily return int, but it won't _not_ work without this conversion, though? Basically, is it really necessary?

Comment (Jojo-Schmitz) [reply mscore/fretproperties.cpp]: it returns a size_t, which is an unsigned type. Alternative woule be to change to `for (size_t i = 0; i < dtypes.size(); i++) {`

Comment (Jojo-Schmitz) [reply mscore/fretproperties.cpp]: "_won't not work without_" too much double negate for me to parse ;-)

Comment (Jojo-Schmitz) [reply mscore/fretproperties.cpp]: Actually:
Return Value
The number of elements in the container.

Member type `size_type` is an unsigned integral type.

(from http://www.cplusplus.com/reference/vector/vector/size/)

Comment (Jojo-Schmitz) [reply mscore/fretproperties.cpp]: Knowing that the vector in question here has just 4 elements makes that cast to int pretty safe though ;-)

Comment (anatoly-os) [reply mscore/fretproperties.cpp]: I would say using 'size_t' with stl containers as a type of the index should become a part of the code style.

Comment (Jojo-Schmitz) [reply mscore/fretproperties.cpp]: OK, next PR ;-)

**Pull request merged**